packages/core/src/server/middlewares/settings-and-store.ts

```diff
@@ -34,5 +34,11 @@ export const settingsAndStore = (packages) => async (
     url: ctx.URL,
   });
 
+  // Populate the context for custom middleware.
+  Object.assign(ctx, {
+    ctx: { ...ctx },
```

Comment:
@luisherranz This is currently broken here, because if I spread ctx, the reference to `ctx.body` is lost, and it cannot be manipulated in both `fn(ctx)` and `fn({ctx})`.